crates/krator-derive/Cargo.toml

```diff
@@ -16,20 +16,37 @@ license-file = "../../LICENSE"
 description = "Derive macros for the Krator crate"
 repository = "https://github.com/deislabs/krustlet"
 readme = "README.md"
-keywords = [
-    "wasm",
-    "wasi",
-    "webassembly",
-    "kubernetes",
-]
+keywords = ["wasm", "wasi", "webassembly", "kubernetes"]
 
 [lib]
 proc-macro = true
-
 [dependencies]
 syn = "1.0"
 quote = "1.0"
 
+anyhow = { version = "1.0.40", optional = true }
```

Comment:
After reading through the new code, I don't think we need most of these new dependencies. The code in the macro is dependent on the project using `krator-derive` to import the correct dependencies

Reply:
yeah ... my hope was that the dependencies get pulled in as a transient dependency. 
Is there a good way to solve this? For example: there is a method creating a secret with a certificate and a private key using the `rcgen` crate. If this is not present, the user of `krator-derive` will get an error that `rcgen` is missing pointing to the derive macro ... is this something that users of macros usually have to solve themselves and is simply documented?

Reply:
I cleaned up the dependecies in commit dec3236

Reply:
Yeah it is generally documented and not pulled into a derive crate. Insofar as I understand things, derive macros are purely meant as transparent code generation